internal/testing/e2e/cluster.go

```diff
@@ -78,22 +79,34 @@ func (com *clusterOM) MustMmLogicGRPC() pb.MmLogicClient {
 	return pb.NewMmLogicClient(conn)
 }
 
-func (com *clusterOM) getGRPCClientFromServiceName(serviceName string) (*grpc.ClientConn, error) {
+func (com *clusterOM) MustMmfConfigGRPC() *pb.FunctionConfig {
+	host, port := com.getAddressFromServiceName("om-e2ematchfunction")
+	return &pb.FunctionConfig{
+		Host: host,
+		Port: port,
+		Type: pb.FunctionConfig_GRPC,
+	}
+}
+
+func (com *clusterOM) getAddressFromServiceName(serviceName string) (string, int32) {
```

Comment:
if this helper was needed only within getGRPCClientFromServiceName - why refactor it out of that method? (unless you plan to reuse it?)

Reply:
This helper is also used by `MustMmfConfigGRPC`